setup.py

```diff
@@ -115,7 +115,8 @@ def run_tests(self):
         'pyyaml',
         'argparse',
         'argcomplete',
-        'future'
+        'future',
+        'pyBarcode==0.8b1'
```

Comment:
Well after looking a little bit more the steenzout fork does not seems very community-friendly as the author of the fork is using his own libraries as requirements. I am more happy to contribute to the viivakoodi project that has no ambiguity about the package name, and seems less one-user-centric.